arangod/VocBase/LogicalView.cpp

```diff
@@ -107,6 +107,14 @@ LogicalView::LogicalView(
   auto const viewType = basics::VelocyPackHelper::getStringRef(
     definition, StaticStrings::DataSourceType, ""
   );
+
+  ///////////////////////////////////////////////////
+  /// Add links checks here:
```

Comment:
this is not a valid comment since view implementations are not required to have a concept of "links"